src/Tests/dotnet-new.Tests/Approvals/DotnetCSharpClassTemplatesTest.struct.langVersion=preview.targetFramework=net7.0.verified/struct/TestItem1.cs

```diff
@@ -0,0 +1,5 @@
+﻿namespace ClassLib;
```

Comment:
It would be good to add the whole project to snapshot. We don't test the project generation, but it would be good to have it for context.